airbyte-integrations/bases/base-normalization/Dockerfile

```diff
@@ -1,4 +1,20 @@
 FROM fishtownanalytics/dbt:0.19.0
+
+USER root
```

Comment:
@ChristopheDuong do you think we need to change again the user here?

Reply:
Not sure, what is the original user?

Reply:
It's root.